### PR TITLE
chore: update versions

### PR DESCRIPTION
**Pull request by NotPeopling2day**
### What I did
Dependabot noted a security issue with cookiecutter, updated to 2.1.1
Updated eth-ape version to 0.3.0
Other small updates

fixes: #3 

### Checklist

- [ ] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [ ] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-TODO: Description
+The ape-template plugin allows you to use cookiecutter to template an ape project.
 
 ## Dependencies
 
-* [python3](https://www.python.org/downloads) version 3.7 or greater, python3-dev
-* [cookiecutter](https://cookiecutter.readthedocs.io/en/2.0.2/) version 1.7.3
-* [eth-ape](https://docs.apeworx.io/ape/stable/) 0.2.2 or greater, 
+* [python3](https://www.python.org/downloads) version 3.7.2 or greater, python3-dev
+* [cookiecutter](https://cookiecutter.readthedocs.io/en/2.0.2/) version 2.1.1
```
Comment (unparalleled-js) [inline README.md:8]: It seems redundant to re-list `install_requires` items (from `setup.py`) here (both `eth-ape` and `cookiecutter`).

File: README.md [hunk 2]
```diff
@@ -30,7 +30,20 @@ python3 setup.py install
...
 ## Quick Usage
 
-TODO: Describe library overview in code
+Use `-h` to list all the commands.
+
+```bash
+ape template -h
+```
+
+Template use example
```
Comment (unparalleled-js) [inline README.md:39]:
```suggestion
To use the `template` command, provide either a GitHub repository ID or a raw URI:
```

File: README.md [hunk 3]
```diff
@@ -30,7 +30,20 @@ python3 setup.py install
...
+
+```bash
+ape template gh:<github org>/<project>
+
+ape template <URI>
+```
```
Comment (unparalleled-js) [inline README.md:45]: Can we have a new line in between code blocks and paragraphs?

File: README.md [hunk 4]
```diff
@@ -30,7 +30,20 @@ python3 setup.py install
...
+[cookiecutter docs](https://cookiecutter.readthedocs.io/en/stable/)
```
Comment (unparalleled-js) [inline README.md:46]:
```suggestion
For more information on Cookiecutter, see their [documentation](https://cookiecutter.readthedocs.io/en/stable/).
```

**Review by unparalleled-js — COMMENTED** [review]
I am finding that I am having to change the commitlint build to only install `commitzen` rather than `ape[dev]` and things are working nicer.

**Comment by unparalleled-js** [comment]
We should also add a simple test, it can even be as simple as making sure the command exists:

```python
def test_help(cli, runner):
    result = runner.invoke("--help")
    assert result.exit_code == 0
```

**Review by fubuloubu — APPROVED** [review]

**Review by unparalleled-js — APPROVED** [review]
Lgtm! Sorry, I had thought there were no tests before, that's why I said to add one

**Comment by Ninjagod1251** [comment]
LGTM :)

**Review by Ninjagod1251 — APPROVED** [review]

**Pull request merged**